frontend/src/components/orders/OfferItem.vue

```diff
@@ -0,0 +1,93 @@
+<script setup lang="ts">
+import Button from "primevue/button";
+import Card from "primevue/card";
+import Dialog from "primevue/dialog";
+import FloatLabel from "primevue/floatlabel";
+import InputNumber from "primevue/inputnumber";
+import InputText from "primevue/inputtext";
+import Textarea from "primevue/textarea";
+import { ref } from "vue";
+
+import { getArticles } from "@/backendClient";
+import { ArticleKind } from "@/global/types/appTypes";
+import type { Article } from "@/global/types/entities";
+
+defineProps<{
+  id: string;
+  type: ArticleKind;
+}>();
+
+let isArticlesListVisible = ref(false);
+const articlesList = ref<Article[]>([]);
+
+async function openArticlesList() {
+  isArticlesListVisible.value = true;
+  articlesList.value = (await getArticles()).data;
+}
+
+let offerInfo = ref({
+  title: "",
+  description: "",
+  number: 0,
+  unit: "",
+  price: 0,
+});
+</script>
+
+<template>
+  <Card class="my-2">
+    <template #content>
+      <div class="flex flex-col gap-y-6">
+        <div class="flex flex-row justify-between">
+          <p class="font-bold">Position {{ id }}</p>
```

Comment:
Is the position (a sequence starting at 1 for each sub-order) really the same as the ID?

Reply:
I will rename it to index since I use it as a position marker.